src/main/kotlin/apply/domain/assignment/Assignment.kt

```diff
@@ -27,7 +27,7 @@ class Assignment(
     @Column(nullable = false)
     var pullRequestUrl: String,
 
-    @Column(length = 1000)
+    @Column(length = 5000)
```

Comment:
```suggestion
    @Column(nullable = false, length = 5000)
```
r: 코틀린의 타입도 non-nullable이고, sql문에서도 `not null`이 있으니 설정을 추가하는게 좋겠습니다!😊

Reply:
예리한 지적 감사합니다.
note는 빈칸이어도 제출을 할 수 있어야 되는 것 같더라고요. sql문에 `not null`을 제거했습니다.